mc-oblivious-ram/src/lib.rs

```diff
@@ -33,11 +33,18 @@ mod position_map;
 pub use position_map::{ORAMU32PositionMap, TrivialPositionMap, U32PositionMapCreator};
 
 mod evictor;
-pub use evictor::{PathOramDeterministicEvictor, PathOramDeterministicEvictorCreator};
-
+pub use evictor::{
+    CircuitOramDeterministicEvictor, CircuitOramDeterministicEvictorCreator,
+    PathOramDeterministicEvictor, PathOramDeterministicEvictorCreator,
+};
 mod path_oram;
 pub use path_oram::PathORAM;
 
+/// For Circuit ORAM to have linear stash size growth at least 2 additional
+/// branches that are sufficiently non overlapping must be evicted every time.
+/// This constant is passed into the evictor creator to specify the number of
+/// additional branches to evict per eviction
+const ADDITIONAL_BRANCHES_TO_EVICT: usize = 2;
```

Comment:
:wrench: This name is generic enough that it's not immediately clear it's only for Circuit ORAM.  The comment helps explain this, but perhaps this should be more narrowly named.
(I know this seems counter to some of my other suggestions to minimize some names, thus is the fickleness of nick)
```suggestion
const CIRCUIT_ORAM_ADDITIONAL_BRANCHES_TO_EVICT: usize = 2;
```

Reply:
This seems reasonable to me.